dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -74,7 +74,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
             THROW_HR_IF_MSG(E_INVALIDARG, m_notificationComActivatorRegistration, "Already Registered for App Notifications!");
             THROW_IF_FAILED(::CoRegisterClassObject(
                 AppModel::Identity::IsPackagedProcess() ? details.TaskClsid() : winrt::guid(storedComActivatorString),
-                winrt::make<AppNotificationActivationCallbackFactory>().get(),
+                winrt::make<AppNotificationManagerFactory>().get(),
```

Comment:
This means that we have two instances of AppNotificationManager. One through Default() and the other created specifically for COM. Can we have just one instance for both?

Reply:
CreateInstance in AppNotificationManagerFactory is not creating a new object but it is returning the same static manager from Default() every time.

Reply:
Excellent!. I had to read forward to understand how this was done. Thanks for this!